### PR TITLE
[tw]Update `Horizontal Scaling` for Traditional Chinese

### LINKED ISSUE #2423: [tw] Localize `Horizontal Scaling` for Traditional Chinese
1. Localize `Horizontal Scaling` for Traditional Chinese
2. https://glossary.cncf.io/horizontal-scaling/

### PR DESCRIPTION
**Pull request by hwchiu**
### Describe your changes
Localize the Horizontal Scaling to zh-tw language
Origin content from https://glossary.cncf.io/horizontal-scaling/

### Related issue number or link (ex: `resolves #issue-number`)
resolves #2423 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by pichuang — APPROVED** [review]
lgtm

File: content/zh-tw/horizontal-scaling.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+title: 水平擴展
+status: Completed
+category: 概念
+tags: ["基礎設施", "", ""]
+---
+
+## 是什麼 {#what-it-is}
+
+水平擴展是一種透過加入更多[節點](/zh-tw/nodes/)而非對單一節點加入更多計算資源來提升系統容量的技術 (後者稱為[垂直擴展](/zh-tw/vertical-scaling/))。
+假設我們有一個 4GB RAM 的系統，並且想要提升其容量到 16 GB RAM，水平擴展代表的是加入 4 * 4GB RAM 的系統而非切換到一台 16GB RAM 的系統。
```
Comment (johnlinp) [inline content/zh-tw/horizontal-scaling.md:11]:
```suggestion
假設我們有一個 4GB RAM 的系統，並且想要提升其容量到 16 GB RAM，水平擴展代表的是加入 4 x 4GB RAM 的系統而非切換到一台 16GB RAM 的系統。
```

File: content/zh-tw/horizontal-scaling.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+我們可以加入更多節點到系統中(水平擴展）或是加入更多計算資源到現有節點上(垂直擴展)
+
+## 如何幫助我們 {#how-it-helps}
+
+水平擴展允許應用程式在底層叢集的限制範圍中去擴展。
+透過加入更多的實例到系統中，應用程式能夠處理更多請求。
+如果單一節點可以每秒處理 1000 個請求，則每一個額外的節點能夠使得每秒總處理請求數量增加大約一千個。
+這使得應用程式可在不加入更多容量到任何節點的情況下去同時執行更多工作。
+
+## 相關詞彙 {$relatead-terms}
```
Comment (johnlinp) [inline content/zh-tw/horizontal-scaling.md:29]:
```suggestion
## 相關詞彙 {#relatead-terms}
```

File: content/zh-tw/horizontal-scaling.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+隨者對應用程式成長的需求超越該應用程式的當前容量，
+我們需要找到一種方式來[擴展](/zh-tw/scalability/)(增加容量)系統。
+我們可以加入更多節點到系統中(水平擴展）或是加入更多計算資源到現有節點上(垂直擴展)
+
+## 如何幫助我們 {#how-it-helps}
+
+水平擴展允許應用程式在底層叢集的限制範圍中去擴展。
+透過加入更多的實例到系統中，應用程式能夠處理更多請求。
+如果單一節點可以每秒處理 1000 個請求，則每一個額外的節點能夠使得每秒總處理請求數量增加大約一千個。
```
Comment (johnlinp) [inline content/zh-tw/horizontal-scaling.md:26]: 句子好像有一點點不通順，想請問如果改成這樣的話會不會好一點：

如果單一節點每秒可以處理 1000 個請求，則每一個額外的節點都能夠讓每秒多處理大約 1000 個請求。

原文：

> If a single node can handle 1,000 requests per second, each additional node should increase the total number of requests by around 1,000 requests per second.

Comment (hwchiu) [reply content/zh-tw/horizontal-scaling.md:26]: 感謝，這樣唸起來的確更通順了，我晚點來修

Comment (hwchiu) [reply content/zh-tw/horizontal-scaling.md]: Updated, please check

**Review by johnlinp — APPROVED** [review]
感謝!

**Pull request merged**